### PR TITLE
[BPF] add -6 to calico-bpf command

### PR DESCRIPTION
**Pull request by tomastigera**
## Description

refs https://github.com/projectcalico/calico/issues/4736

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
TBD
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

File: felix/cmd/calico-bpf/commands/conntrack.go
Comment (sridhartigera) [inline felix/cmd/calico-bpf/commands/conntrack.go:498]: conntrack.Key?

**Review by sridhartigera — APPROVED** [review]
Just one comment. Otherwise LGTM

**Pull request merged**
